src/AutoMapper/composer.json

```diff
@@ -20,7 +20,7 @@
     "require-dev": {
         "doctrine/annotations": "~1.0",
         "phpdocumentor/reflection-docblock": "^3.0 || ^4.0",
-        "symfony/serializer": "^4.2 || 5.0.*"
+        "symfony/serializer": "^4.2 || ^5.0"
```

Comment:
This composer file should also conflict framework-bundle 5.1.0

Reply:
my bad, I'll add it to every package, that requires sf serializer